src/client/automation/playback/click/index.js

```diff
@@ -212,6 +215,10 @@ export default class ClickAutomation extends VisibleElementAutomation {
                 return Promise.all([delay(this.automationSettings.mouseActionStepDelay), this._mousedown(eventArgs)]);
             })
             .then(() => this._mouseup(eventArgs))
-            .then(() => this._click(eventArgs));
+            .then(({ timeStamp }) => {
+                eventArgs.options.timeStamp = timeStamp;
```

Comment:
Does `mousedown` event should have `timeStamp` property value same as for `mouseup` and `click`?

Reply:
no, we can hold mouse button